docs/topics/Gateway.md

```diff
@@ -645,7 +656,7 @@ A user's presence is their current state on a guild. This event is sent when a u
 | guild_id | snowflake | id of the guild |
 | status | string | either "idle", "dnd", "online", or "offline" |
 
-### Game Object
+#### Game Object
```

Comment:
I added this because all of the other H3s in this section are gateway events. I didn't want people to think GAME_OBJECT is an event based only on the heading style, so I "indented" it.